static/js/src/advantage/react/components/Subscriptions/SubscriptionDetails/SubscriptionDetails.tsx

```diff
@@ -136,7 +136,14 @@ export const SubscriptionDetails = forwardRef<HTMLDivElement, Props>(
                   >
                     Edit subscription&hellip;
                   </Button>
-                ) : null}
+                ) : (
```

Comment:
Will this not make it so that any subscription that is not `isResizable` will show the message: "This subscription cannot be edited online"? One problem is that `legacy` user subscriptions have `isResizable` always `false` because we do not allow them to be resized. If we want this to show up only for `Offers`, we need to condition it by user subscription type.

Reply:
Or did we drop the idea of having user subscriptions of type `offer`?

Reply:
Sorry I left some comments on the ticket but not here. Since we can't differentiate an offer from a legacy without extra API calls, I asked Will if we could just do a generic message whenever we are dealing with something we can't edit/resize. The messaging was updated on [miro](https://miro.com/app/board/o9J_lydRtns=/?moveToWidget=3458764514679240808&cot=14) now too.

Reply:
I see.. but my first comment still stands. If we cannot differentiate between `legacy` and `offers` then it mean we cannot correctly condition for the message to show only for `offers`. 

The condition for displaying this should still not be based on the `else` side of `isResizable`. 

Can CS not help us with distinguishing `legacy` from `offers`?